File: UBL/CommonAggregateComponents/LocationCoordinateType.cs
using Hilma.UBL.Attributes;
using Hilma.UBL.UnqualifiedDataTypes;

namespace Hilma.UBL.CommonAggregateComponents
{
  [Contract]
  public class LocationCoordinateType
  {

    public CodeType CoordinateSystemCode { get; set; }

    public MeasureType LatitudeDegreesMeasure { get; set; }

    public MeasureType LatitudeMinutesMeasure { get; set; }

    public CodeType LatitudeDirectionCode { get; set; }

    public MeasureType LongitudeDegreesMeasure { get; set; }

    public MeasureType LongitudeMinutesMeasure { get; set; }

    public CodeType LongitudeDirectionCode { get; set; }

    public MeasureType AltitudeMeasure { get; set; }

  }
}

File: Espd/typings/LocationCoordinateType.ts



import { CodeType } from './CodeType'
import { MeasureType } from './MeasureType'

import assign from 'lodash.assign'

export class LocationCoordinateType { 
    public constructor(init?:Partial<LocationCoordinateType>) {
        assign(this, init)
    } 
    
    
    coordinateSystemCode?: CodeType
    latitudeDegreesMeasure?: MeasureType
    latitudeMinutesMeasure?: MeasureType
    latitudeDirectionCode?: CodeType
    longitudeDegreesMeasure?: MeasureType
    longitudeMinutesMeasure?: MeasureType
    longitudeDirectionCode?: CodeType
    altitudeMeasure?: MeasureType
}